cmd/dagger/main.go

```diff
@@ -181,6 +185,7 @@ func installGlobalFlags(flags *pflag.FlagSet) {
 	flags.BoolVarP(&debug, "debug", "d", debug, "Show debug logs and full verbosity")
 	flags.StringVar(&progress, "progress", "auto", "Progress output format (auto, plain, tty)")
 	flags.BoolVarP(&interactive, "interactive", "i", false, "Spawn a terminal on container exec failure")
+	flags.StringVar(&interactiveCommand, "interactive-command", "/bin/sh", "Change the default command for interactive mode")
```

Comment:
Sorry for getting to the party late!

Potentially, how would you feel about having this merged into the existing `interactive` command? (we already do this with `--compat` for example). That way, we could have `--interactive` go to the default, and `--interactive="foo"` execute command `foo`. The one downside is that you *can't* do `--interactive "foo"`, you need to do it all in one arg with `=`.

Up to you, just offering another suggestion!

Reply:
I wasn't aware of the `--compat` one, interesting. I'd like to keep this way because 90% of the time, users will only use `--interactive`, and it's more intuitive from the help to just have a boolean command. With your suggestion, I would instead default to --interactive="", meaning the help would show `--interactive string[=""]`.